test/test_visualization.jl

```diff
@@ -3,6 +3,7 @@ module TestVisualization
 using Test
 using Trixi
 using Plots
+using Makie
```

Comment:
Oops - this should be GLMakie. That's why the test is failing.

I'll wait until @sloede is done with his review before updating.